src/ui/menu/menuView.tsx

```diff
@@ -193,6 +193,11 @@ function renderLinks(user?: Session) {
         <span className="fa fa-at"/>{i18n('players')}
       </li> : null
       }
+      {online ?
```

Comment:
I think it's still not ready to be released. So let's remove it from the menu for now. I should add a project with a todo list of what needs to be done before release, asap.

Reply:
Ok, just removed.